unlock-app/src/components/interface/checkout/main/Confirm.tsx

```diff
@@ -172,13 +231,18 @@ export function Confirm({
     data: purchaseData!,
     paywallConfig,
     enabled: !isInitialDataLoading,
+    symbol: lockTickerSymbol(
+      lock as Lock,
+      config.networks[lock!.network].nativeCurrency.symbol
+    ),
   })
 
   const isPricingDataAvailable =
     !isPricingDataLoading && !isPricingDataError && !!pricingData
 
   const amountToConvert = pricingData?.total || 0
 
+  // TOFIX: @searchableguy This does not seem accurate when there are multiple recipients...
   const { data: totalPricing, isInitialLoading: isTotalPricingDataLoading } =
```

Comment:
@searchableguy Please check but I suspect this won't work well when there are multiple recipients...

Reply:
I will re-check but this simply gets the total amount in native currency after applying discounts and then use that to get fiat charges and total.

Reply:
You are correct! I was confused with the name here. Fixing it https://github.com/unlock-protocol/unlock/pull/11687